naptime-graphql/src/main/scala/org/coursera/naptime/ari/graphql/controllers/GraphQLController.scala

```diff
@@ -118,7 +118,10 @@ class GraphQLController @Inject() (
       variables: JsObject,
       operation: Option[String]): Future[OutgoingQuery] = {
     Future {
-      QueryParser.parse(query) match {
+      val parsedQuery = metricsCollector.timeQueryParsing {
+        QueryParser.parse(query)
+      }
```

Comment:
This is a start.  Do you think the later assignments to `incomingQuery`, `filterFn` and the call to `filterFn(incomingQuery)` are worth tracking, too?

Reply:
If this doesn't give us any actionable data, we can add some more tracking in. I'm pretty sure those are all very cheap operations, so I don't want to over-collect data. But it's easy enough to add it in later.

Reply:
agreed